src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -70,60 +69,12 @@ class PaymentMethodList extends Component {
      * @returns {Array}
      */
     createPaymentMethodList() {
-        const combinedPaymentMethods = [];
-
-        _.each(this.props.bankAccountList, (bankAccount) => {
-            // Add all bank accounts besides the wallet
-            if (bankAccount.type === CONST.BANK_ACCOUNT_TYPES.WALLET) {
-                return;
-            }
-
-            const formattedBankAccountNumber = bankAccount.accountNumber
-                ? `${this.props.translate('paymentMethodList.accountLastFour')} ${
-                    bankAccount.accountNumber.slice(-4)
-                }`
-                : null;
-            const {icon, iconSize} = getBankIcon(lodashGet(bankAccount, 'additionalData.bankName', ''));
-            combinedPaymentMethods.push({
-                type: MENU_ITEM,
-                title: bankAccount.addressName,
-
-                // eslint-disable-next-line
-                description: formattedBankAccountNumber,
-                icon,
-                iconSize,
-                onPress: e => this.props.onPress(e, bankAccount.bankAccountID),
-                key: `bankAccount-${bankAccount.bankAccountID}`,
-            });
-        });
-
-        _.each(this.props.cardList, (card) => {
-            const formattedCardNumber = card.cardNumber
-                ? `${this.props.translate('paymentMethodList.cardLastFour')} ${card.cardNumber.slice(-4)}`
-                : null;
-            const {icon, iconSize} = getBankIcon(card.bank, true);
-            combinedPaymentMethods.push({
-                type: MENU_ITEM,
-                title: card.addressName,
-                // eslint-disable-next-line
-                description: formattedCardNumber,
-                icon,
-                iconSize,
-                onPress: e => this.props.onPress(e, card.cardNumber),
-                key: `card-${card.cardNumber}`,
-            });
-        });
-
-        if (this.props.payPalMeUsername) {
-            combinedPaymentMethods.push({
-                type: MENU_ITEM,
-                title: 'PayPal.me',
-                description: this.props.payPalMeUsername,
-                icon: Expensicons.PayPal,
-                onPress: e => this.props.onPress(e, 'payPalMe'),
-                key: 'payPalMePaymentMethod',
-            });
-        }
+        let combinedPaymentMethods = PaymentUtils.getPaymentMethods(this.props.bankAccountList, this.props.cardList, this.props.payPalMeUsername);
+        combinedPaymentMethods = _.map(combinedPaymentMethods, paymentMethod => ({
+            ...paymentMethod,
+            type: MENU_ITEM,
```

Comment:
What's the point in giving these all `type` params and then overwriting it with `MENU_ITEM`? Do we need a type to be returned from `getPaymentMethods()`?

Reply:
Oh yeah. Sorry, Just got confused from the previous PR. I will change it